crates/neon-runtime/src/napi/error.rs

```diff
@@ -9,3 +16,25 @@ pub unsafe extern "C" fn new_type_error(_out: &mut Local, _msg: Local) { unimple
 pub unsafe extern "C" fn new_range_error(_out: &mut Local, _msg: Local) { unimplemented!() }
 
 pub unsafe extern "C" fn throw_error_from_utf8(_msg: *const u8, _len: i32) { unimplemented!() }
+
+pub unsafe extern "C" fn is_throwing(env: Env) -> bool {
+    let mut b: MaybeUninit<bool> = MaybeUninit::zeroed();
```

Comment:
FYI, Since `bool` has a trivial uninitialized state, this could also be `let mut b = false;`

Reply:
I think it's better to use `MaybeUninit` to document that we are expecting the N-API call to do the initializing.